modules/databricks-loader/src/main/scala/com/snowplowanalytics/snowplow/loader/databricks/Databricks.scala

```diff
@@ -62,7 +63,8 @@ object Databricks {
 
           override def initQuery[F[_]: DAO: Monad]: F[Unit] = Monad[F].unit
 
-          override def createTable(schemas: SchemaList): Block = Block(Nil, Nil, Entity.Table(tgt.schema, schemas.latest.schemaKey))
+          override def createTable(shredModel: ShredModel): Block =
+            Block(Nil, Nil, Entity.Table(tgt.schema, shredModel.schemaKey, shredModel.tableName))
```

Comment:
Am I right that this `createTable` should never get called for Databricks?  Or for Snowflake?

If yes, I think I'd prefer to see it implemented as:

```scala
override def createTable(shredModel: ShredModel): Block =
  throw new IllegalStateException("createTable should never be called for Databricks")
```

Otherwise, I find it confusing to see it returns an object using `shredModel.tableName`.